### ISSUE TITLE
[Enhancement]: Upload files directly into folders

### ISSUE THREAD

**Issue by PulsarFTW** [advplyr/audiobookshelf#1984]
### Describe the feature/enhancement

When uploading a file, open a file browser (server side folders) so that uploads can be sent directly into a chosen folder. 

I realise the upload feature is not really used and understand if this is a low priority, however if the feature isn't planned to be discontinued then this change would make the feature more useful.

Thanks!

**Comment by edsilberberg** [comment]
Agreed. It could be an /import local mount point for manual ingestion.
If it was manual it could be in the upper portion of the /audiobookshelf/upload page for server side uploads, with a distinction for client side uploads.

<img width="2050" height="795" alt="Image" src="https://github.com/user-attachments/assets/8f4c0fde-73c3-4cd9-88f8-8e426d7bd11f" />


Or get fancy and do a even a shelfmark > calibre web automated style auto ingestion folder similar to what what asked for in https://github.com/advplyr/audiobookshelf/issues/2594

I have suggested to @coconautilus17 the writer of Libraforge (which is a fantastic abs metadata tool worth checking out) to create a staging directory. That would be a great target for auto ingest. https://github.com/coconautilus17/LibraForge
